### PR TITLE
Use webview instead

### LINKED ISSUE #9: Update app to use new page
Keycloak was updated. It's now dsek.se. Make sure the web version of the app works.

### LINKED ISSUE #10: ✨ Add correct notification channelIds for android version of app

### PR DESCRIPTION
**Pull request by Macludde**
This replaces the ENTIRE app to only use a Webview and scrap everything else.

It still has support for notifications and login etc.

### CODE REVIEW

**Comment by Macludde** [comment]
Closes #9 and #10

File: providers/NotificationProvider.tsx [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
 import useNotifications from '~/hooks/useNotifications';
-import { NavigationProp, useNavigation } from '@react-navigation/native';
-import { NewsStackParamList } from '~/types/navigation';
+import WebView from 'react-native-webview';
 
-const NotificationProvider: React.FC = ({ children }) => {
+const NotificationProvider: React.FC<{ webref: Ref<typeof WebView> }> = ({ webref }) => {
   const token = useNotifications();
-  const navigation = useNavigation<NavigationProp<NewsStackParamList>>();
+  console.log(token)
```
Comment (danieladugyan) [inline providers/NotificationProvider.tsx]:
```suggestion
```

File: components/MainView.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      };
+    }
+  }, []);
+
+  return (
+    <SafeAreaView style={{ flex: 1, backgroundColor: '#121212' }}>
+      <NotificationProvider webref={webViewRef} />
+      <WebView
+        source={{ uri: WEBSITE_URL }}
+        // allowsBackForwardNavigationGestures /* for swipe navigation on iOS */
```
Comment (danieladugyan) [inline components/MainView.tsx]: Uncomment/delete?

**Review by danieladugyan — APPROVED** [review]
Some minor details, but overall, LGTM!

**Pull request merged**